ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.19.yaml

```diff
@@ -63,7 +63,7 @@ releases:
       product: ocp
       relative: 1
       stream: ci
-      version: "4.19"
+      version: "4.18"
```

Comment:
`ci-operator/config/openshift-priv` is automatically sync'ed from `ci-operator/config/openshift` on a regular basis.  This change is the only one that is really required.